src/services/db/GitHubService.ts

```diff
@@ -190,7 +239,13 @@ export default class GitHubService {
     return { content, sha }
   }
 
-  async readMedia(sessionData, { fileSha, branchName = STAGING_BRANCH }) {
+  async readMedia(
+    sessionData: { accessToken?: any; siteName?: any },
```

Comment:
Should this be `UserWithSiteSessionData`?

Reply:
b7fa30138bb1b5bacb5ff10b367e6c6d0d81473e